tests/teams/createTeam.test.ts

```diff
@@ -145,8 +174,36 @@ describe('POST /teams', () => {
 
     await request(app)
       .post('/teams')
-      .send({ name: 'otherName', tournamentId: teamBody.tournamentId })
+      .send({ name: 'otherName', tournamentId: teamBody.tournamentId, captainType: teamBody.captainType })
       .set('Authorization', `Bearer ${otherToken}`)
       .expect(410, { error: Error.TournamentFull });
   });
+
+  it('shall deny orga captain type', async () => {
```

Comment:
```suggestion
  it('should deny orga captain type', async () => {
```
Tu préfères être plus formel ? 😁
Perso, je garderais le `should` histoire de garder une cohérence entre tous les tests ^^.